src/options/file_name.rs

```diff
@@ -108,3 +113,23 @@ impl EmbedHyperlinks {
         }
     }
 }
+
+impl Absolute {
+    fn deduce(matches: &MatchedFlags<'_>) -> Result<Self, OptionsError> {
+        let mode_opt = matches.get(&flags::ABSOLUTE)?;
+        if !matches.has(&flags::ABSOLUTE)? && mode_opt.is_none() {
+            return Ok(Self::Nothing);
```

Comment:
Nitpick but unuseful condition there. (I know this is how its done on the other parts of the code but I am trying, slowly, to remove that type of code)